<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>softdev</class>
 <widget class="QMainWindow" name="softdev">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>softdev</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #39bef7;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">border-radius: 2px</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1141</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>1081</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="homePg">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>80</y>
          <width>321</width>
          <height>311</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 40px</string>
        </property>
        <property name="text">
         <string>HOMEPAGE DUDS</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="projPg">
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>0</y>
          <width>841</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="plainText">
         <string>Search here</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="projScroll">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>40</y>
          <width>851</width>
          <height>561</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: gray</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>851</width>
           <height>561</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>0</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>0</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QPushButton" name="settingsBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>520</y>
          <width>221</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white</string>
        </property>
        <property name="text">
         <string>SETTINGS</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addProjBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>440</y>
          <width>221</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#a8a8a8;</string>
        </property>
        <property name="text">
         <string>ADD PROJECT +</string>
        </property>
       </widget>
       <widget class="QPushButton" name="editProfBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>360</y>
          <width>221</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:gray;</string>
        </property>
        <property name="text">
         <string>EDIT PROFILE</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>220</y>
          <width>221</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string/>
        </property>
        <property name="text">
         <string>Profile Area</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>221</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 30px;</string>
        </property>
        <property name="text">
         <string>PROJECTS</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="compPg">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>90</y>
          <width>491</width>
          <height>331</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 40px;</string>
        </property>
        <property name="text">
         <string>COMPETITION REDACTEDS</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="settings">
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>70</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 15px;</string>
        </property>
        <property name="text">
         <string>Desktop notifications</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>0</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 33px;</string>
        </property>
        <property name="text">
         <string>SETTINGS</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>170</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 15px;</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>200</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 15px;</string>
        </property>
        <property name="text">
         <string>Number of Projects</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>230</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 15px;</string>
        </property>
        <property name="text">
         <string>Interests</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>260</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 15px;</string>
        </property>
        <property name="text">
         <string>School</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>100</y>
          <width>261</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 19px;</string>
        </property>
        <property name="text">
         <string>WHAT OTHERS CAN SEE</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="addProj"/>
      <widget class="QWidget" name="editProfile"/>
     </widget>
     <widget class="QPushButton" name="projBtn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>PROJECTS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="homeBtn">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>HOME</string>
      </property>
     </widget>
     <widget class="QPushButton" name="compBtn">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>20</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>COMPETITION</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>530</y>
        <width>391</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
font-size: 20px</string>
      </property>
      <property name="text">
       <string>LOG IN</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="userInput">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>220</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
font-size: 15px</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:15px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:15px;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="passInput">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>370</y>
        <width>451</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
font-size: 15px</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:15px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:15px;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
